packages/by-name/microsoft/kata-igvm/package.nix

```diff
@@ -4,53 +4,68 @@
 {
   lib,
   stdenv,
+  stdenvNoCC,
   microsoft,
   igvm-tooling,
   igvm-signing-keygen,
+  igvmmeasure,
 }:
 
-stdenv.mkDerivation rec {
-  pname = "kata-igvm";
-  inherit (microsoft.genpolicy) src version;
+let
+  igvm = stdenv.mkDerivation rec {
```

Comment:
What's the point of moving this into a let binding?

Reply:
Allows access to the attribute in passthru, as let bindings are recursive. But I guess we could also access it through 
`microsoft.igvm` from the input attrs.